### PR TITLE
new feature and reverted old changes

### PR DESCRIPTION
**Pull request by harisalam**
Kindly merge these files into the main repository with changed data clean function, error fixed while connecting to mysql and some dependency issues

### CODE REVIEW

File: src/service.py [hunk 1]
```diff
@@ -135,9 +138,9 @@ def cluster_sentences(self, language_name: str, save_path: str, sentences: List[
 
     # first loading udpipe to segement word for each sentence
     udt_english = UdpipeTrain(language_list[1],
-                              '/home/zglg/SLU/psd/pre-model/english-ewt-ud-2.5-191206.udpipe',
```
Comment (hvkone) [inline src/service.py:138]: We need to change this section. Please allow user to select the file to input.

File: src/util.py [hunk 2]
```diff
@@ -11,10 +11,8 @@
 db_config = {'user': 'root',
              'password': 'root',
              'db_host': 'localhost',
-
-             'db_name': 'psdProject'}
-
              'db_name': 'psd_project'}
-cluster_model_file = {'Chinese': '/home/zglg/SLU/psd/cluster_pre_train/gensim-word2vec-model-Chinese',
-                      'English': '/home/zglg/SLU/psd/cluster_pre_train/gensim-word2vec-model-English'}
+
```
Comment (hvkone) [inline src/util.py:15]: We need to add a user file input function here, and everywhere else that calls a file.

**Review by hvkone — CHANGES_REQUESTED** [review]
Please consider using user file input for calling specific files.

**Comment by harisalam** [comment]
You can merge the request everything is in good shape, I've double checked everything

**Comment by hvkone** [comment]
working on user file input function

**Pull request merged**